src/main/resources/db/migration/V1_56_1__remove_appointment_fields_from_action_plan.sql

```diff
@@ -0,0 +1,21 @@
+create table action_plan_session (
+    id uuid not null,
+    session_number int not null,
+    action_plan_id uuid,
+    constraint fk_action_plan_session_action_plan foreign key (action_plan_id) references action_plan,
+    primary key (id)
+);
+
+create table action_plan_session_appointments (
+    action_plan_session_id uuid not null,
+    appointments_id uuid not null,
```

Comment:
2 observations
- it sounds like this relationship could be represented in a simpler way with an `action_plan_appointment_v2 { action_plan_id, session_number, appointment_id }` table
- given we won't ever drop sessions later, just add more, we have the option to link these to `referral` instead of the action plan, essentially giving us: `referral_session_appointment { referral_id, session_number, appointment_id }`

Reply:
so we would have 

```
appointment with pk id: UUID

action_plan with pk id: UUID

action_plan_session with pk (action_plan_id: UUID, session_number: Int)

action_plan_appointment with pk (appointment_id: UUID)
```

?

Reply:
was thinking maybe
```
appointment with pk id: UUID

action_plan with pk id: UUID

delivery_appointment with pk (action_plan_id: UUID, session_number: Int)
  field FK (appointment_id: UUID)
```
?

Reply:
if you flesh this out to support multiple appointments per session i think you kind of end up with what we have here? an extra join table which ends up being an appointment 'container' - i.e. the session?

Reply:
i suppose you can just create multiple of the thing you're calling 'delivery_appointment' with the same session number and action plan id?

Reply:
talked offline -- all good 👍 I forgot we have to put delivery method etc somewhere too, and they'll go on the session-related bits

Reply:
discussed offline. leaving as is for now.